### PR TITLE
Feat: added landing page

### PR DESCRIPTION
**Pull request by anjali1102**
https://user-images.githubusercontent.com/56559378/154859270-201367ca-d7ba-44a2-ad6b-0c3b5744db74.mp4

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]

File: style/style.css [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+
+:root {
+  --body-font: "Open Sans", sans-serif;
+  --big-font-size: 3rem;
+  --h1-font-size: 1.5rem;
+  --h2-font-size: 1.25rem;
+  --normal-font-size: 0.938rem;
+  --small-font-size: 0.75rem;
+}
+
```
Comment (jeetsdev) [inline style/style.css:28]: You can combine all these in a single `:root` selector...

File: style/style.css [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+:root {
+  --z-index: 10;
+  --z-fixed: 100;
+}
+
+*,
+::before,
+::after {
+  box-sizing: border-box;
+}
```
Comment (jeetsdev) [inline style/style.css:55]: If you are using your component lib, then why a reset style CSS here?

File: style/style.css [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+  padding-top: 6rem;
+  flex: 1;
+}
+/* .home-title {
+  align-self: center;
+  font-size: var(--big-font-size);
+  line-height: 1;
+  letter-spacing: 0.5rem;
+  margin-bottom: 4rem;
+} */
```
Comment (jeetsdev) [inline style/style.css:210]: Please remove unnecessary code.

**Review by SK02K1 — APPROVED** [review]

File: style/style.css [hunk 4]
```diff
@@ -0,0 +1,423 @@
...
+  .nav__toggle {
+    display: none;
+  }
+
+  .home__container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    height: 100vh;
+    /* height: auto; */
```
Comment (jeetsdev) [inline style/style.css:367]: Remove unnecessary code.

**Review by jeetsdev — CHANGES_REQUESTED** [review]
Added some suggestions...

**Review by jeetsdev — APPROVED** [review]
Looks good now 💯

**Pull request merged**